packages/grid_client/src/helpers/utils.ts

```diff
@@ -49,4 +49,18 @@ function toHexSeed(mnemonicOrHexSeed: string): string {
   return "0x" + seed;
 }
 
-export { generateString, getRandomNumber, randomChoice, randomSecret, randomSecretAsHex, randomNonce, log, toHexSeed };
+function formatErrorMessage(prefix: string, error: Error) {
+  return `${prefix}\n\tdue to: ${error.message}`;
+}
```

Comment:
After ending each error with a dot, this format might not be good. Can we change it to be like that?
```suggestion
  return `${prefix}\n\t ${error.message}`;
}
```